app/src/main/res/drawable/start_over_button_background.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners android:radius="@dimen/state_previous_next_button_radius" />
+  <stroke
+    android:width="4px"
```

Comment:
Why is this `px`? Normally we keep all values as `dp`

Reply:
done